.github/actions/deploy-my-kibana/action.yml

```diff
@@ -55,6 +58,19 @@ runs:
 
         # issue_comment does not contain any references to github.base_ref
         TARGET_BRANCH=$(gh pr view ${PR} --repo ${{ inputs.repository }} --json baseRefName --jq .baseRefName)
+
+        # If serverless
+        if [ "${{ inputs.serverless }}" == 'true' ] ; then
+          if [ "${TARGET_BRANCH}" == 'main' ] ; then
+            # As long as serverless is not yet available we use stateless instead
+            #echo "CLUSTER=serverless-oblt" >> $GITHUB_ENV
+            echo "CLUSTER=stateless-oblt" >> $GITHUB_ENV
```

Comment:
We use `stateless` for now, as agreed with @kuisathaverat. We will eventually replace it, but both serverless and stateless are related to the `serverless` but serverless is not user available yet.